docs/manage/enabling_surveys_using_config_files.md

```diff
@@ -0,0 +1,241 @@
+
+6. For each button you wish to include in your Study/Program, add a key object as a child of `"buttons"`. The name you give your button object MUST match that of the `key-name` in the table above.
+
+```json
+"buttons": {
+    "key-name1": {
+        //HERE
+    },
+    "key-name2": {
+        //HERE
+    }
+}
+```
+
+7. The keys contained inside of each survey object are **important**. Here is the template to follow, and we will explain what each field is for:
+
+```json
+"key-name1": {
+    "surveyName": "",
+    "not-filled-in-label": {
+        "en": "",
+        "es": ""
+    },
+    "filled-in-label": {
+        "en": "",
+        "es": ""
+    }
+}
+```
+
+- **`"surveyName":`** defines the survey that will display when this button is clicked
+- **`"not-filled-in-label":`** contains the internationalizations for the text displayed on the button when the survey **has not** been filled out yet.
+  -  **`"en":` / `"es":`/ etc.** contains the text which will be displayed on the button when the survey **has not** been filled out yet. For example, this might read `"Take my survey"`
+- **`filled-in-label":`** contains the internationalizations for the text displayed on the button when the survey **has** been filled out.
+  -  **`"en":` / `"es":`/ etc.** contains the text which will be displayed on the button when the survey **has** been filled out. For example, this might read `"Response submitted"`
+
+8. Finally, you will define how the trip-labels (e.g. the labels for mode and purpose of each trip) will be collected. There are only 2 options currently supported by OpenPATH: `MULTILABEL` and `ENKETO`.
+
+```json
+...
+"trip-labels": "MULTILABEL"
+...
+```
+or
+```json
+...
+"trip-labels": "ENKETO"
+...
+```
+
+9. Now that your config file has been completed, you can compare it to some of our examples in the section below _Examples of configs with varying surveys (for reference)_ to make sure you have written it correctly.
+
+# Wrap up:
+
+1. Publish your changes to your fork of  `nrel-openpath-deploy-configs`.
+
+2. Start the emulator and uninstall/reinstall your `em-devapp.app` so that it opens up with the config login screen and prompts you to scan into a Study/Program.
+
+3. Open Safari in the emulator and paste in your `emission://join_study?label=YOUR_STUDY_OR_PROGRAM_LABEL&source=github`, and it should automatically open up e-mission and your config will successfully be loaded into the app with the study/program you've created. _If you are using an emulator with Google Chrome, this will **NOT** work. You must click on the link, by starting up the docker-compose of [`nrel-openpath-deploy-configs`](https://github.com/sebastianbarry/nrel-openpath-deploy-configs/tree/surveys-info-and-surveys-data) and navigating to `http://[your local IP address]:9090/` to see a list of available links found in the `https://github.com/sebastianbarry/nrel-openpath-deploy-configs/blob/surveys-info-and-surveys-data/survey-resources/index.js`_
```

Comment:
```suggestion
3. Open Safari in the emulator and paste in your `emission://join_study?label=YOUR_STUDY_OR_PROGRAM_LABEL&source=github`, and it should automatically open up e-mission and your config will successfully be loaded into the app with the study/program you've created. _If you are using an emulator with Google Chrome, this will **NOT** work. You must click on the link, by starting up the docker-compose of [`nrel-openpath-deploy-configs`](https://github.com/sebastianbarry/nrel-openpath-deploy-configs/tree/surveys-info-and-surveys-data) and navigating to `http://10.0.2.2:9090/` (10.0.2.2 is the Android shortcut for "localhost") to see a list of available links found in the `https://github.com/sebastianbarry/nrel-openpath-deploy-configs/blob/surveys-info-and-surveys-data/survey-resources/index.js`_
```